pkg/database/user_report_test.go

```diff
@@ -106,10 +106,11 @@ func TestFindUserReport(t *testing.T) {
 			t.Parallel()
 
 			userReport, err := db.NewUserReport(tc.phone, []byte{0}, true)
-			userReport.Nonce = tc.nonce // override the encoding from NewUserReport for testing errors.
 			if err != nil {
 				t.Fatalf("error creating user report: %v", err)
 			}
+
+			userReport.Nonce = tc.nonce // override the encoding from NewUserReport for testing errors.
```

Comment:
Unrelated, but fixes a bug. userReport is nil if there's an error, so setting the nonce will fail.